drivers/usb/device/Kconfig

```diff
@@ -55,29 +55,29 @@ config USB_DC_SAM
 	help
 	  SAM family USB HS device controller Driver.
 
-menuconfig USB_NRF52840
-	bool "Nordic Semiconductor nRF52840 USB Device Controller Driver"
+menuconfig USB_NRF52
```

Comment:
Could we rename it to just USB_NRF5 (I guess nRF5340 will have the same USB IP)?

Reply:
@jfischer-phytec-iot USB_NRF5 is also not perfect because then it will mean that Nordic could not update the USB chip in any future cores in nrf5x series... I would rather keep this name, do you agree?

Reply:
@barsok it makes no real sense to keep this name, since nrf5340 does have the same USB IP; we need a name that covers both nrf52 and nrf53. I like USB_NRF5 but if you have a better suggestion, pls, shoot.

Reply:
I am fine with USB_NRFX. https://github.com/zephyrproject-rtos/zephyr/pull/20222#pullrequestreview-313333666